spec/rubocop/cli/auto_gen_config_spec.rb

```diff
@@ -430,6 +430,50 @@ def f
             Max: 95
         YAML
       end
+
+      it 'can generate a todo list if default .rubocop.yml exists' do
+        create_file('example1.rb', ['def foo', '  # bar', '  end'])
+        create_file('.rubocop.yml', <<~YAML)
+          AllCops:
+            DisabledByDefault: true
+
+          Layout/DefEndAlignment:
+            Enabled: true
+        YAML
+        create_file('cop_config.yml', <<~YAML)
+        YAML
```

Comment:
Can you use `create_empty_file` instead?

```suggestion
        create_empty_file('cop_config.yml')
```

Reply:
Thank you, I've committed and squashed it.